conbench/app/_plots.py

```diff
@@ -450,11 +607,7 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
         bokeh.models.HoverTool(
             tooltips=[
                 ("date", "$x{%F}"),
-                # Note(JP): this is where the `means` name becomes special,
-                # I think.
-                ("mean", "@means"),
-                ("commit", "@commit_hashes_short"),
-                ("commit msg", "@commit_messages"),
+                ("value", "@values_with_unit"),
```

Comment:
Now the word "value" appears in the tooltip where before it said "mean".

The `value_with_unit` list contains value-with-unit strings, one item is for example `"1.2 s"` So, this effectively says to show something like `value: 1.2 s` in the tooltip.

Reply:
What is the thinking for this change? It _is_ the mean right? My inclination would be to actually expand it and match it to "benchmark mean" to align language with the legend.

Reply:
Great question. These code sections are rather hard to see through, and I am slowly changing things in hopefully digestible chunks.

It's not always the mean. The same function as of today builds various Bokeh `ColumnDataSource`s, including for single sample benchmarks (n=1) and also for displaying min and max values for multi-sample benchmarks, and even for the 'alert min' and 'alert max' time series which seem to be the 5sigma band boundaries.

We should try to tidy this up step by step, not changing everything at once, but in controllable pieces.

Right now we still have code like

```
"mean": benchmark["stats"]["min"]
```
(wow)

and also
```
points.append(float(x["distribution_mean"]) + alert)
```

And in these cases we previously added the stringified-value-with-unit as `"mean", "@means"` to the `ColumnDataSource`. To make the code a tiny less misleading: that's why I changes this to the more general concept of `"value", "@values_with_unit"` here.

In other words, right now `_source()` is a generic function that is called for various y axis units and value meanings. 

The function is not too important anyway, we can move on from here and I am super OK with this being gone or looking completely different maybe already next week.